packages/frontity-org-theme/src/processors/background-color.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { css } from "frontity";
+import { Processor } from "@frontity/html2react/types";
+import FrontityOrg from "../../types";
+
+const colorClassRegex = /has-(\w+)-background-color/;
```

Comment:
@SantosGuillamot can you confirm that color names never contain other characters than letters and numbers? I.e.: `light-grey` is not possible. If that's not the case, this regexp should include the allowed characters.

By the way, is the list of allowed colors documented somewhere?

Reply:
They could contain dashes if we add them (or if we uses spaces), but it something we are not using right now. Maybe is a good idea to support that in the regexp just in case. What do you think @DAreRodz ?

It's not documented yet (I'm planning to prepare a guide with all the colors, classes and custom group blocks), but they are defined in both the [WordPress side](https://wp.frontity.org/wp-admin/admin.php?page=edit-snippet&id=8&result=updated) and the [Frontity state](https://github.com/frontity/frontity.org/blob/dev/packages/frontity-org-theme/src/index.ts#L11-L21). Right now, we have to make sure that they match, but maybe in the future we can expose the palette colors and fill the state.

Apart from that, inside the Gutenberg block editor, you can easily select all the colors included in the palette.